custom_components/mopidy/config_flow.py

```diff
@@ -105,40 +106,44 @@ async def async_step_user(self, user_input=None):
 
     async def async_step_zeroconf(self, discovery_info: DiscoveryInfoType):
         """Handle zeroconf discovery."""
-        try:
-            await self.hass.async_add_executor_job(
-                _validate_input, discovery_info["hostname"], discovery_info["port"]
-            )
-        except reConnectionError:
-            _LOGGER.warning(
-                "%s@%d is not a mopidy server",
-                discovery_info["hostname"],
-                discovery_info["port"],
-            )
-            return self.async_abort(reason="not_mopidy")
-        except:  # noqa: E722 # pylint: disable=bare-except
-            _LOGGER.error(
-                "An error occurred connecting to %s:%s",
-                discovery_info["hostname"],
-                discovery_info["port"],
-            )
-            return self.async_abort(reason="not_mopidy")
+        if discovery_info[CONF_TYPE] == '_mopidy-http._tcp.local.':
```

Comment:
This inferred by the entry in `manifest.json`, if I'm not mistaken.
Only zeroconf devices with this `CONF_TYPE` are passed onto the component.

Reply:
I'm not really sure to understand, you mean we don't need this check right?

Reply:
Fixed, thanks!